src/pages/ChargerMap/ChargerMap.js

```diff
@@ -80,6 +80,8 @@ export default function ChargerMap(props) {
           let description = e.features[0].properties.description;
           let combined = coordinates +'<br />' + sname +'<br />' + description  +'<br />' + provider;
 
+          setLat(e.lngLat.lat);
+          setLng(e.lngLat.lng);
```

Comment:
Consider defensive coding, us the optional chaining. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining